pkg/syncer/reconcile/as_unstructured.go

```diff
@@ -56,5 +57,6 @@ func AsUnstructuredSanitized(o client.Object) (*unstructured.Unstructured, statu
 
 	unstructured.RemoveNestedField(u.Object, "metadata", "creationTimestamp")
 	unstructured.RemoveNestedField(u.Object, "status")
+	unstructured.RemoveNestedField(u.Object, "metadata", "managedFields")
```

Comment:
The `managedFields` field is tracked by the Kubernetes API server. Custom controllers should not manage this field.